content/cloud/pricing-and-billing.md

```diff
@@ -15,23 +15,23 @@ url = "https://github.com/fermyon/developer/blob/main//content/cloud/payment-and
 - [Next Steps](#next-steps)
 
 ## Fermyon Cloud Pricing
-Fermyon Cloud offers subscription plans to help you increase the limits of your Fermyon Cloud usage. This page covers general information about subscription plans and answers frequently asked questions around [payment](#payment), [refunds](#refund), [support](#support), and [quota](#quota).
+Fermyon Cloud offers subscription plans to help you increase the limits of your Fermyon Cloud usage. This page covers general information about subscription plans and answers frequently asked questions around [payment](#payment), [refunds](#refund), [support](#support), and [quotas](#quota).
```

Comment:
I'm not sure I'm seeing any change to this?

Reply:
I think it's because the comment preview didn't refresh. Here is the latest verbiage:

> Fermyon Cloud offers subscription plans to help you accomplish your workload goals which require an increase Fermyon Cloud usage. 

If that sounds good to you, I'll resolve this comment!